src/cromshell/__main__.py

```diff
@@ -53,7 +53,8 @@
     help="Highest level logging for debugging",
 )
 @click.option(
-    "--hide_logo",
+    "--no_turtle",
+    "--I_hate_turtles",
```

Comment:
What?!

Reply:
https://github.com/broadinstitute/cromshell/pull/221#discussion_r1018345749

Reply:
sigh.... poor turtles

Reply:
🐢

Reply:
Only a monster doesn't display the turtle.

Reply:
🐢